src/rpc/core_rpc_server.cpp

```diff
@@ -616,6 +616,23 @@ namespace cryptonote
       return true;
     }
 
+    unsigned int concurrency_count = std::thread::hardware_concurrency() * 4;
```

Comment:
Another botched comment.
@moneromooo-monero I am slowly looking at all thread usage, and will slowly be making tweaks as previously discussed.